force-app/main/default/lwc/nksConversationNoteDetails/nksConversationNoteDetails.js

```diff
@@ -51,6 +60,43 @@ export default class NksConversationNoteDetails extends LightningElement {
         }
     }
 
+    @wire(getReverseRelatedRecord, {
+        parentId: '$recordId',
+        queryFields: 'Account.CRM_Person__c',
+        objectApiName: 'Case',
+        relationshipField: 'Id'
+    })
+    wierdRelatedRecord({ error, data }) {
+        if (data && data.length > 0) {
+            this.personId = this.resolve('Account.CRM_Person__c', data[0]);
+        } else if (error) {
+            console.error(error);
+        }
+    }
+
+    @wire(getRecord, {
+        recordId: '$personId',
+        fields: PERSON_ACTORID_FIELD
+    })
+    wiredPersonInfo({ error, data }) {
+        if (data) {
+            const actorId = getFieldValue(data, PERSON_ACTORID_FIELD);
+            this.readAccessToPerson = actorId ? true : false;
+        } else if (error) {
+            console.error(error);
+        }
+    }
+
+    resolve(path, obj) {
+        if (typeof path !== 'string') {
+            throw new Error('Path must be a string');
+        }
+
+        return path.split('.').reduce(function (prev, curr) {
+            return prev ? prev[curr] : null;
+        }, obj || {});
+    }
```

Comment:
Kan denne funksjonen flyttes til nksButtonContainerUtils for å unngå gjentakelse av funksjonen i forskjellige komponenter/repoer? Vi kan rename funksjonen til for eks. nksComponentUtils og importere resolve funksjonen i de forskjellige komponentene :)